### PR TITLE
Create dreamweaver-light.css

### PR DESCRIPTION
**Pull request by adrianTNT**

### CODE REVIEW

**Comment by yyyc514** [comment]
Snapshot?

**Comment by adrianTNT** [comment]
58109_747x667

File: src/styles/dreamweaver-light.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
+/* inspired by Dreamweaver code colors, created by https://www.TNTcode.com */
+
+.hljs {
+  display: block;
+  overflow-x: auto;
+  padding: 0.5em;
+  background: #fff;
+  font-family: "Courier New", monospace;
+  font-size: 13px;
```
Comment (yyyc514) [inline src/styles/dreamweaver-light.css]: Don't embellish please, no font family or sizing (keep it the same as all the other themes).

**Comment by yyyc514** [comment]
Also please add copyright and license.  Example:

https://github.com/highlightjs/highlight.js/pull/2151/files

**Comment by yyyc514** [comment]
Closing due to inactivity.

**Pull request closed without merging**